### PR TITLE
Reconcile CSV mapper docs

### PR DESCRIPTION
**Pull request by diana-macrometa**
Multiple entries for CSV sink and source mapper exist. This PR attempts to reconcile them and remove the redundant one. 

Reviewers, please look at the comments for questions and also closely review the example output. I think some formatting is missing or incorrect.

### CODE REVIEW

File: docs/cep/query-guide/functions/sourcemapper/csv.md [hunk 1]
```diff
@@ -2,42 +2,43 @@
...
-| event.grouping.enabled    | This parameter specifies whether event grouping is enabled or not. To receive a group of events together and generate multiple events, this parameter must be set to `true`.                                                                                                                                                                           | false         | BOOL                | Yes      | No      |
+| delimiter                 | When converting a CSV format message to a stream worker event, this parameter indicates input CSV message's data should be split by this parameter.   | ,             | STRING              | Yes      | No      |
+| header.present            | When converting a CSV format message to Stream App event, this parameter indicates whether CSV message has header or not. This can either have value `true` or `false`. If it's set to `false`, then it indicates that CSV message has no header.   | false         | BOOL                | Yes      | No      |
```
Comment (diana-macrometa) [inline docs/cep/query-guide/functions/sourcemapper/csv.md:19]: @StoyanTc and @ashish-macrometa - This parameter was not present in the reference version of the doc. Is it correct?

**Comment by diana-macrometa** [comment]
@StoyanTc or @ashish-macrometa, will you be able to review this soon?

File: docs/cep/query-guide/functions/sourcemapper/csv.md [hunk 2]
```diff
@@ -2,42 +2,43 @@
...
+Above configuration will do a default CSV input mapping. Expected input is:
+
+gdn ,55.6 , 100OR  "gdn,No10,Palam Groove Rd,Col-03" ,55.6 , 100
+
+If `header.present` is `true` and delimiter is "-",
+then the input is as follows: `symbol-price-volumegdn-55.6-100`
 
 ## Example 2
 
     CREATE SOURCE FooStream WITH (type='inMemory', topic='stock', map.type='csv', header='true', map.attributes="symbol = '2', price = '0', volume = '1'") (symbol string, price float, volume long);
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/sourcemapper/csv.md]: The syntax of the `example 2` has issue, the correct one is
```
CREATE SOURCE FooStream WITH (type='stream', topic='stock', map.type='csv', map.header='true', map.attributes.symbol = '2', map.attributes.price = '0', map.attributes.volume = '1') (symbol string, price float, volume long);
```

File: docs/cep/query-guide/functions/sourcemapper/csv.md (hunk 1 above)
Comment (StoyanTc) [reply docs/cep/query-guide/functions/sourcemapper/csv.md:19]: @diana-macrometa - I guess you mean `header.present`? Yes, it's valid.

File: docs/cep/query-guide/functions/sourcemapper/csv.md [hunk 3]
```diff
@@ -2,42 +2,43 @@
...
+| header.present            | When converting a CSV format message to Stream App event, this parameter indicates whether CSV message has header or not. This can either have value `true` or `false`. If it's set to `false`, then it indicates that CSV message has no header.   | false         | BOOL                | Yes      | No      |
+| fail.on.unknown.attribute | This parameter specifies how unknown attributes should be handled. If it's set to `true` and one or more attributes don't have values, then the stream processor will drop that message. If this parameter is set to `false`, the stream processor adds the required attribute values to such events with a null value and the event is converted to a stream worker event. | true          | BOOL                | Yes      | No      |
+| event.grouping.enabled    | This parameter specifies whether event grouping is enabled or not. To receive a group of events together and generate multiple events, this parameter must be set to `true`. | false         | BOOL                | Yes      | No      |
 
 ## Example 1
 
     CREATE SOURCE FooStream WITH (type='inMemory', topic='stock', map.type='csv') (symbol string, price float, volume int);
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/sourcemapper/csv.md]: Let's replace `inMemory` by `stream`.

File: docs/cep/query-guide/functions/sinkmapper/csv.md [hunk 4]
```diff
@@ -21,9 +20,9 @@ Syntax
 
     CREATE SINK BarStream WITH (type='inMemory', topic='{{symbol}}', map.type='csv') (symbol string, price float, volume long);
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/sinkmapper/csv.md]: Let's replace `inMemory` by `stream`.

**Review by StoyanTc — APPROVED** [review]
Added few comments, the rest is good.

**Pull request merged**